### PR TITLE
Adding article about ACM and ODF on ARO

### PR DESCRIPTION
**Pull request by ricmmartins**

### CODE REVIEW

File: content/aro/acm-odf-aro/index.md [hunk 1]
```diff
@@ -0,0 +1,1135 @@
...
+  --vnet-name "$AZR_CLUSTER-aro-vnet-$AZR_RESOURCE_LOCATION" 	\
+  --name JumpSubnet                                          	\
+  --address-prefixes $JUMPHOST_SUBNET    
+  ```
+
+2. Create a jump host
+
+``` az vm create --name jumphost                 		\
+    --resource-group $AZR_RESOURCE_GROUP     	        \
+    --ssh-key-values $HOME/.ssh/id_rsa.pub   	        \
```
Comment (kmcolli) [inline content/aro/acm-odf-aro/index.md]: Change to an environment variable.

File: content/aro/acm-odf-aro/index.md [hunk 2]
```diff
@@ -0,0 +1,1135 @@
...
+
+echo $JUMP_IP
+```
+
+4. Use sshuttle to create a SSH VPN via the jump host (use a separate terminal session)
+
+Replace the IP with the IP of the jump box from the previous step
+
+```
+sshuttle --dns -NHr "aro@${JUMP_IP}"  10.0.0.0/8
```
Comment (kmcolli) [inline content/aro/acm-odf-aro/index.md:190]: Change 10.0.0.0/8 to an environment variable .. for example I have a conflict with 10.0.x so am using 10.1.x

Comment (kmcolli) [reply content/aro/acm-odf-aro/index.md:190]: Running the above command with sshuttle doesn't allow you to run the create cluster command.  Add steps to run the sshuttle in the background.   Also, do we really need to run sshuttle at this point?

**Comment by theckang** [comment]
Side note: If we switch the UI steps to CLI steps, I think we should cherry pick those commits into main.  We do not want to pollute the main repo with screenshot files that we end up deleting, because it will bloat the repo size and increases the time it takes to pull/fetch from main.

**Comment by theckang** [comment]
We are converting UI steps to CLI and will cherrypick the unnecessary UI files out of this.  Will submit PR when ready.

File: content/aro/acm-odf-aro/index.md [hunk 3]
```diff
@@ -0,0 +1,1135 @@
...
+
+3. Save the jump host public IP address
+
+```JUMP_IP=$(az vm list-ip-addresses -g $AZR_RESOURCE_GROUP -n jumphost -o tsv \
+--query '[].virtualMachine.network.publicIpAddresses[0].ipAddress')
+
+echo $JUMP_IP
+```
+
+4. Use sshuttle to create a SSH VPN via the jump host (use a separate terminal session)
```
Comment (kmcolli) [inline content/aro/acm-odf-aro/index.md:185]: if you open a new terminal - you lose the JUMP_IP environment variable

**Pull request closed without merging**